Samples/Desktop/D3D12Raytracing/src/D3D12RaytracingHelloWorld/D3D12RaytracingHelloWorld.cpp

```diff
@@ -381,6 +412,17 @@ void D3D12RaytracingHelloWorld::BuildAccelerationStructures()
         AllocateUAVBuffer(device, topLevelPrebuildInfo.ResultDataMaxSizeInBytes, &m_topLevelAccelerationStructure, initialResourceState, L"TopLevelAccelerationStructure");
     }
 
+    // Note on Emulated GPU pointers (AKA Wrapped pointers) requirement in Fallback Layer:
```

Comment:
@wallisc This is a copied excerpt from FL's developer guide. The emulated pointers documentation should be more explicit where the emulated pointers are required. i.e. only for bottom level and top level acceleration structures, because the concept is clear but where everywhere it applies it is not.